### PR TITLE
Move docs from aicoe-cd to cd

### PR DESCRIPTION
**Pull request by HumairAK**
Tried to generalize the docs so it's not aicoe-cd specific. Closes #17 .

### CODE REVIEW

File: docs/create_argocd_application_manifest.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Application Management
+
+While ArgoCD allows you to create ArgoCD applications via the UI and CLI, it is our
```
Comment (anishasthana) [inline docs/create_argocd_application_manifest.md]: What do you think of `it is our policy` -> `we recommend`

File: docs/create_argocd_application_manifest.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+policy that all applications be created [declaratively](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#applications).
+
+This allows us to easily restore your applications should the need arise.
```
Comment (anishasthana) [inline docs/create_argocd_application_manifest.md]: `us` -> `you`

File: docs/create_argocd_application_manifest.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+## Pre-requisites
+- Kustomize version 3.8+
+
+## Steps for creating an application
+For your application to show up to ArgoCD you need to do 2 things:
+1. Create the Application yaml in the appropriate path in a fork
+2. Submit a PR to this repository
```
Comment (anishasthana) [inline docs/create_argocd_application_manifest.md]: `to the base repository`

File: docs/create_argocd_application_manifest.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+- `spec.destination.cluster`: the target cluster server name for this `Application`'s deployment, ensure this server exists in  `manifests/overlays/<target_env>/secrets/clusters` for the appropriate cluster.
+
+- `spec.source.path`: path to the Kustomization.yaml file relative to the repo's root.
+
+- `spec.source.repoURL`: the repository holding the `Application`'s desired state, ensure this repo exists within `argo_cm` located in `manifests/overlays/<target_env>/configs/argo_cm/repositories`.
+
+  In case you are adding a private repo:
+
+  1. Please ensure [@sesheta](https://github.com/sesheta) bot has access to your repo (directly or indirectly via [@AICoE/sourceops](https://github.com/orgs/AICoE/teams/sourceops)/[@aicoe-aiops/sourceops](https://github.com/orgs/aicoe-aiops/teams/sourceops) teams)
```
Comment (anishasthana) [inline docs/create_argocd_application_manifest.md]: We don't need this right?

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: docs/create_argocd_application_manifest.md (hunk 4 above)
Comment (durandom) [reply docs/create_argocd_application_manifest.md]: Yes, `sourceops` seems to be AICoE specific. Either rework to be generic for private repos, or just drop. (and bullet point 2.)

**Review by durandom — CHANGES_REQUESTED** [review]
LGTM.

Let's address the minor suggestions and go for it.
I'd love to see our first downstream (aicoe-cd) to be a fork of this repo soon :)

**Pull request merged**